src/clang.rs

```diff
@@ -448,7 +453,11 @@ impl Cursor {
 
     /// Get the visibility of this cursor's referent.
     pub fn visibility(&self) -> CXVisibilityKind {
-        unsafe { clang_getCursorVisibility(self.x) }
+        if clang_getCursorVisibility::is_loaded() {
+            unsafe { clang_getCursorVisibility(self.x) }
+        } else {
+            CXVisibility_Default
```

Comment:
Oh, yeah, those are mostly because we don't have the field offset, that's pretty good actually :)